toolchain/lower/file_context.h

```diff
@@ -219,6 +294,34 @@ class FileContext {
 
   // Global format string for `printf.int.format` used by the PrintInt builtin.
   llvm::Value* printf_int_format_string_ = nullptr;
+
+  // For a generic function, keep track of the specifics for which LLVM
+  // function declarations were created. Those can be retrieved then via
+  // from specific_functions_, via specific_functions_[specific_id.index].
+  // We resize this to the correct size. Indexed by generic_id.index.
```

Comment:
It'd be helpful for me if this could be phrased similarly to what's already done above, for example on `specific_functions_` and `functions_`. Particularly for this detail, I think it's helpful to keep a consistent way of referring to it. Referring to the type helps me because there's not a `generic_id` value in this context.

Note, I'm adding a suggestion here, but there's similar below on other structs -- this is just an example of how I'd suggest fixing it.

```suggestion
  // We resize this to the correct size. Vector indexes correspond to `GenericId` indexes.
```